compiler/src/iree/compiler/Dialect/Stream/Transforms/EncodeTensors.cpp

```diff
@@ -335,18 +347,39 @@ static Value canonicalizeFillPattern(Value pattern, PatternRewriter &rewriter) {
 
   // Get floats into integer form.
   auto patternType = pattern.getType();
-  unsigned bitWidth = patternType.getIntOrFloatBitWidth();
+  unsigned elemBitWidth = getTypeBitwidth(patternType);
```

Comment:
```suggestion
  unsigned elementBitWidth = getTypeBitwidth(patternType);
```
prefer full words to abbreviations (abbreviations are harder for non-native english speakers to understand)